source/FreeRTOS_TCP_Reception.c

```diff
@@ -331,14 +331,14 @@
                          */
                         if( pucPtr[ 0U ] == tcpTCP_OPT_SACK_A )
                         {
-                            ucLen -= 2U;
+                            ucLen -= 2;
                             lIndex += 2;
 
                             while( ucLen >= ( uint8_t ) 8U )
                             {
                                 prvReadSackOption( pucPtr, ( size_t ) lIndex, pxSocket );
                                 lIndex += 8;
-                                ucLen -= 8U;
+                                ucLen -= 8;
```

Comment:
This is not solving the issue . Earlier warning was "**conversion from ‘unsigned int’ to ‘uint8_t**’" with your fix the warning has now changed to **conversion from ‘int’ to ‘uint8_t’**